src/config/scrivito.js

```diff
@@ -15,3 +15,9 @@ if (process.env.SCRIVITO_PRERENDER) {
 }
 
 Scrivito.configure(config);
+
+Scrivito.configureObjClassForContentType({
+  "image/*": "Image", // Factory default
```

Comment:
* [ ] FYI not sure if the "Factory default" comments add value here, or are more confusing than helpful. For a user of the Example App the factory default is this file. How about dropping the comments?